### PR TITLE
onItemClick prop added to contextual menu

### PR DESCRIPTION
**Pull request by notgiorgi**
#### Pull request checklist

- [x] Addresses an existing issue: #1141 
- [x] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [x] Documentation update

#### Description of changes

See #1141

### CODE REVIEW

**Comment by notgiorgi** [comment]
I triggered CI build again, because it failed because of unsuccessful nodejs download (weird)

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx [hunk 1]
```diff
@@ -450,15 +450,18 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
...
     this._executeItemClick(item, ev);
     ev.stopPropagation();
   }
 
-  private _executeItemClick(item: any, ev: MouseEvent) {
+  private _executeItemClick(item: IContextualMenuItem, ev: React.MouseEvent<HTMLElement>) {
     if (item.onClick) {
       item.onClick(ev, item);
+    } else {
+      this.props.onItemClick(ev, item);
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx]: What is `onItemClick` is null/undefined?

File: common/changes/master_2017-03-16-08-24.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "`onItemClick` prop added to ContextualMenu",
```
Comment (cschleiden) [inline common/changes/master_2017-03-16-08-24.json]: nit, can you format it `ContextualMenu: .. description..`

**Comment by notgiorgi** [comment]
Fixed up things according to the review

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts [hunk 3]
```diff
@@ -103,6 +103,12 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
   onDismiss?: (ev?: any, dismissAll?: boolean) => void;
 
   /**
+   * Click handler which is invoked if onClick is not passed for individual contextual
+   * menu item
+   */
+  onItemClick?: (ev?: React.MouseEvent<HTMLElement>, item?: IContextualMenuItem) => void;
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts]: Order of your props seems inconsistent. What is standard, event first, or item? Seems in the component we were always putting item first.

Comment (notgiorgi) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts]: Yep, I skimmed through the docs and most of the onSometing handlers are passing item first (not all of the though). I'll refactor it

**Comment by notgiorgi** [comment]
Is this still being reviewed?

**Comment by micahgodbolt** [comment]
@cschleiden are you good with this now? I'm ready to merge.

**Pull request merged**